app/src/main/kotlin/xyz/duckee/android/DuckeeApplication.kt

```diff
@@ -19,10 +19,22 @@ import android.app.Application
 import coil.ImageLoader
 import coil.ImageLoaderFactory
 import dagger.hilt.android.HiltAndroidApp
+import co.ab180.airbridge.Airbridge
+import co.ab180.airbridge.AirbridgeOption
+import co.ab180.airbridge.AirbridgeOptionBuilder
 
 @HiltAndroidApp
 class DuckeeApplication : Application(), ImageLoaderFactory {
 
+    override fun onCreate() {
+        super.onCreate()
+
+        // Initialize Airbridge SDK
+        val option = AirbridgeOptionBuilder("undefined", "undefined")
+            .build()
+        Airbridge.initializeSDK(this, option)
```

Comment:
Added Airbridge SDK initialization. This is necessary to integrate Airbridge functionality into the app. The `AirbridgeOptionBuilder` is used with placeholder values for the app name and SDK token. These should be replaced with actual values from the Airbridge dashboard before deploying the app. As per the Airbridge SDK documentation, initialization should occur in the `Application` class's `onCreate` method to ensure proper functionality.